addons/website/static/src/components/configurator/configurator.js

```diff
@@ -485,6 +504,18 @@ async function makeEnvironment() {
     await router.start();
     const state = await getInitialState();
     const store = new Store({state, actions, getters});
+    store.on("update", null, () => {
```

Comment:
what about store it in a the # part of url ?  

and since features is the last step, why save it ?

Reply:
> what is the advantage of doing that ?

nothing sotred until the end if your session, easy to copy-past url...

So yes, I will just keep the 3 fields from first step, and the palette, feature_id. So just 5 info in # part of url no?

Reply:
lgtm

Reply:
After investigation and a discussion with Géry, dynamic paths is not well supported by Owl router (see the issue I just submitted: https://github.com/odoo/owl/issues/858). It is supported only for a specific format.

Navigation of the configurator is implemented using Owl router. So we won't be able to use the # part of the url to store data (at least until owl support it). If we really want to use the # part instead of sessionStorage we will have to throw away the current navigation implementation and use something else.

Imo we should use session storage and implement a solution with the # part of the url when owl support it.

Reply:
I didn't succeed to use Owl's router to put info the # part of the url (an other problem: https://github.com/odoo/owl/issues/863). Moreover I realized that we also have to store the user uploaded logo. So I decided to implement a solution using sessionstorage.

Reply:
I think that you dont need to keep logo, only color ;)
but if owl always doesnt works... lets move with it... waiting that owl support this basic case

Reply:
The logo is not only used for the colors. It is also set as the website's logo in `configurator_apply` so if we loose it during a page refresh, it won't be set on the website. That's the reason why I think it is better to keep it.